avm/res/network/load-balancer/README.md

```diff
@@ -0,0 +1,1193 @@
+# Load Balancers `[Microsoft.Network/loadBalancers]`
+
+This module deploys a Load Balancer.
+
+## Navigation
+
+- [Resource Types](#Resource-Types)
+- [Usage examples](#Usage-examples)
+- [Parameters](#Parameters)
+- [Outputs](#Outputs)
+- [Cross-referenced modules](#Cross-referenced-modules)
```

Comment:
The [notes](https://github.com/Azure/ResourceModules/tree/main/modules/network/load-balancer) of the original Readme got lost. Was that intentional?

If not, I'd recommend to copy the readme from CARML and re-run the readme generation (which should preserve the notes).